cli/command/context/import.go

```diff
@@ -0,0 +1,69 @@
+package context
+
+import (
+	"errors"
+	"fmt"
+	"io"
+	"os"
+
+	"github.com/docker/cli/cli"
+	"github.com/docker/cli/cli/command"
+	"github.com/docker/cli/cli/context/store"
+	"github.com/spf13/cobra"
+)
+
+type importOptions struct {
+	use   bool
+	force bool
+	name  string
+}
+
+func newImportCommand(dockerCli command.Cli) *cobra.Command {
+	opts := importOptions{}
+	cmd := &cobra.Command{
+		Use:   "import [OPTIONS] FILE",
```

Comment:
This should be the mirror of the export command `import [OPTIONS] FILE [CONTEXT]`

Reply:
I really hate commands with 2 unnamed args of same type (string) but completely different meaning. I can never remember the order of args (e.g. I never ever write `ln -s` in the correct way on the first try), that is why I originally wrote it like this. But I understand the need for mirroring.
My preferred alternate would be `docker context export CONTEXT --to FILE` and `docker context import CONTEXT --from FILE` but that does not feel aligned with the rest of the cli design.